test/ApiCheckBaseline.V1/ComparisonScenarios.cs

```diff
@@ -52,4 +52,9 @@ public class ClassWithMethods
     public class TypeWithExtraInterface
     {
     }
+
+    public interface IInterfaceToAddMembersTo
```

Comment:
Can we add another scenario for `IInterfaceMemberRenamed`? and another one where the new one has members removed (so old one has more?)